variables.tf

```diff
@@ -191,3 +191,13 @@ variable "service_credential_names" {
     error_message = "The specified service credential role is not valid. The following values are valid for service credential roles: 'Writer', 'Reader', 'Manager'"
   }
 }
+
+variable "metrics" {
+  type        = list(string)
+  description = "Enhanced metrics to activate, as list of strings. Allowed values: 'topic', 'partition', 'consumers'."
```

Comment:
If this is only valid for enterprise plan instances, can we mention this in the description here? We should probably also add validation to the module to fail if a user tried to pass a value here if the plan is not enterprise

Reply:
Added to the description, and also added a check to main.tf.  Output of the check:
```
│   on ../../main.tf line 28, in locals:
│   28:   validate_metrics = var.plan != "enterprise-3nodes-2tb" && length(var.metrics) > 0 ? tobool("metrics are only supported for enterprise plan") : true
│     ├────────────────
│     │ while calling tobool(v)
│ 
│ Invalid value for "v" parameter: cannot convert "metrics are only supported for enterprise plan" to bool; only the strings "true" or "false" are
│ allowed.
```